<ui version="4.0" >
 <class>NormalEstimationDialog</class>
 <widget class="QDialog" name="NormalEstimationDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Normal estimation</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Direction:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="m_inputDirection" >
     <item>
      <property name="text" >
       <string>quadric</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>plane</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QSpinBox" name="m_inputNbNeighborsDirection" >
     <property name="suffix" >
      <string> neighbors</string>
     </property>
     <property name="minimum" >
      <number>6</number>
     </property>
     <property name="maximum" >
      <number>9999</number>
     </property>
     <property name="value" >
      <number>18</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Orientation:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QComboBox" name="m_inputOrientation" >
     <item>
      <property name="text" >
       <string>MST</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QSpinBox" name="m_inputNbNeighborsOrientation" >
     <property name="suffix" >
      <string> neighbors</string>
     </property>
     <property name="minimum" >
      <number>6</number>
     </property>
     <property name="maximum" >
      <number>9999</number>
     </property>
     <property name="value" >
      <number>18</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NormalEstimationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NormalEstimationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
